build.sbt

```diff
@@ -1,30 +1,55 @@
-enablePlugins(ScalaJSPlugin)
+// enablePlugins(ScalaJSPlugin)
 
-enablePlugins(ScalaJSBundlerPlugin)
+// enablePlugins(ScalaJSBundlerPlugin)
 
 // @TODO[Security] Is this a good idea to leave this here long term?
 // resolvers += Resolver.sonatypeRepo("snapshots")
 
-libraryDependencies ++= Seq(
-  "com.raquo" %%% "airstream" % "0.10.0",
-  "com.raquo" %%% "domtypes" % "0.10.1",
-  "com.raquo" %%% "domtestutils" % "0.12.0" % Test,
-  "org.scalatest" %%% "scalatest" % "3.1.1" % Test
-)
 
-scalacOptions ++= Seq("-deprecation", "-feature", "-language:higherKinds", "-language:implicitConversions")
+// lazy val core = (project )
 
-version in installJsdom := "16.2.0"
+lazy val jsdocs = project
+  .settings(
+   libraryDependencies += "org.scala-js" %%% "scalajs-dom" % "1.1.0"
+  )
+  .enablePlugins(ScalaJSPlugin)
+  .dependsOn(laminar)
 
-useYarn := true
+lazy val docs = project       // new documentation project
+  .in(file("myproject-docs")) // important: it must not be docs/
+  .settings(
+    moduleName := "site-docs"
+  )
+  .enablePlugins(MdocPlugin, DocusaurusPlugin)
+  .settings(
+    mdocJS := Some(jsdocs)
+  )
 
-requireJsDomEnv in Test := true
+lazy val laminar = project.in(file("."))
```

Comment:
Does it make sense to move laminar code to a `laminar` subfolder?

Reply:
I'll defer to @raquo's preference on this one - technically there's no need to do this as it's still a single module and a single deliverable, and I can imagine that moving all the sources in this PR would make it more daunting  :)

It would make more sense if Airstream wasn't a separate library and was just a module here, which would probably make releasing cascade simpler, and means the docs could include both airstream and laminar examples

Reply:
Hmm for now let's keep Laminar in `.`, and I'll take a look at this again when reviewing all this at the end.

Moving Airstream into this git repo sounds compelling, but I'm not sure how to deal with git version tags in that case since Laminar version doesn't always match Airstream version, and git tags are currently the primary navigation mechanism for looking at the code or docs for a particular version.